Modules/QuestieTracker.lua

```diff
@@ -200,7 +200,7 @@ local function _ShowQuestLog(Quest)
             QuestLogExFrameMaximizeButton:GetScript("OnClick")(QuestLogExFrameMaximizeButton)
         end
     else
-        QuestLogFrame:Show()
+       ToggleQuestLog();
```

Comment:
```suggestion
        ToggleQuestLog();
```

Reply:
Such an easy fix, thanks for this!

Reply:
Anytime! Glad to contribute to something so useful, i'll be around more now that i finally hit 60.

Reply:
Haha, sounds good! We appreciate any help we get. @AeroScripts and I are around level 30 😅